docker-testing/src/images/mod.rs

```diff
@@ -0,0 +1,62 @@
+use std::time::Duration;
+
+use rand::{distributions::Alphanumeric, Rng};
+
+mod bitcoin_core;
+mod dynamodb;
+mod emily;
+
+// Bitcoin core types.
+pub use bitcoin_core::BitcoinCore;
+pub use dynamodb::DynamoDb;
+pub use emily::Emily;
+
+pub const DEFAULT_BITCOIN_CORE_TAG: &str = "28";
+
+/// Generates a unique container name with an 8-character random alphanumeric suffix.
+///
+/// ## Parameters
+/// * `name` - Base name for the container
+///
+/// ## Returns
+/// A string formatted as "sbtc-test-{name}-{random_suffix}"
+fn container_name(name: &str) -> String {
+    let suffix: String = rand::thread_rng()
+        .sample_iter(&Alphanumeric)
+        .take(12)
+        .map(char::from)
+        .collect();
+
+    format!("sbtc-test-{name}-{suffix}")
```

Comment:
Could we include the test name in the container name for easier debugging?  

- Current: `sbtc-test-bitcoind-abc123`  
- Maybe: `sbtc-test-bitcoind-test_get_block_works-abc123`?

In this example, `test_get_block_works` is the test name (can probably be derived).

Reply:
Using a derive macro, sure, but otherwise we're stuck either providing the name as a string, or using a (questionably reliable) hack like `any::type_name<fn()>()` and stripping module prefix/suffixes...